c/include/ml-api-service.h

```diff
@@ -117,16 +117,78 @@ int ml_service_delete_pipeline (const char *name);
 
 /**
  * @brief handle for ml_service. To support multiple instance for a single service, user should mangage pipeline with this handle.
+ * @since_tizen 7.0
  */
 typedef void *ml_service_h;
 
-/** @todo Add description and decide final API name */
+/**
+ * @brief Launches the pipeline of given service and get the handle.
+ * @details This requests machine learning agent daemon to launch a new pipeline of given service. The pipeline of service @a name should be set.
+ * @since_tizen 7.0
+ * @remarks The @a handle should be destroyed using ml_serice_destroy_pipeline().
```

Comment:
typo in function name, or maybe you meant ml_service_destroy() which destroys pipeline?

Reply:
I fixed it to `ml_service_destroy`. Thanks.